update the README file (#4)
ansible-collections/vmware: README.md

Ansible Collection: community.vmware
====================================

> This is the now official place where VMware modules are maintained.

Reviewer: ```suggestion
The Official place where VMware Community modules are maintained.
```
